### ISSUE TITLE
What the heck are phases?

### ISSUE THREAD

**Issue by Customery** [MicrosoftDocs/powerapps-docs#3687]
The guidance offered here is an example of an outdated linear, sequential approach to building business apps in phases.

This isn't 1984 any more.

We're agile now. We don't have phases. We concurrently analyse, design, develop, test and deploy our application in increments that we frequently inspect and adapt. 

I hope your documentation can keep up.

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3772f8af-9355-92ac-2bd2-77016dac0ca2
* Version Independent ID: 30c0cc3b-003f-c67f-9d4f-3dc81328f511
* Content: [Introduction to Planning a Power Apps project - Power Apps](https://learn.microsoft.com/en-us/power-apps/guidance/planning/introduction)
* Content Source: [powerapps-docs/guidance/planning/introduction.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/guidance/planning/introduction.md)
* Service: **powerapps**
* Sub-service: **guidance**
* GitHub Login: @TGrounds
* Microsoft Alias: **thground**

**Comment by bthompson-365** [comment]
I have to agree with Neil here because what is written here doesn't reflect any of the diagrams on https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/guidance/planning/app-development-approaches.md which this page links to.

Personally the whole point of developing power apps is because it allows you to bypass the Conceptial / prototype phase and instead build a simple MVP  (minimum viable product) which can be given to users who can then provide feedback regarding it's viability and possible enhancements.

This page seems to fit in a previous era of waterfall based project management very much at odds with how most power platform development is done.

**Comment by bishalgoswami** [comment]
Thanks for your comment, @Customery. This issue has been assigned to the doc author for review.
